.rubocop.yml

```
@@ -0,0 +1,28 @@
+
+Layout/TrailingWhitespace:
+  Enabled: true
+
+Style/MethodDefParentheses:
+  Enabled: true
+
+Style/StringLiterals:
+  EnforcedStyle: single_quotes
+
+Style/StringLiteralsInInterpolation:
+  EnforcedStyle: single_quotes
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)